tests/helpers/asyncio_helper.py

```diff
@@ -0,0 +1,114 @@
+"""
+Helper module for asyncio methods
+"""
+import os
+import sys
+import asyncio
+import logging
+import helpers.filter_message_helper
+import helpers.sqs_helper
+import conf.channel_configuration_conf as channel_conf
+import conf.sqs_utilities_conf as queue_url_conf
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+#Defining current file path
+CURR_FILE_PATH = os.path.dirname(os.path.abspath(__file__))
+MESSAGES_PATH = os.path.join(CURR_FILE_PATH, '')
+CULTURE_FILE = os.path.join(MESSAGES_PATH, 'culture.txt')
+
+#setting environment variable
+os.environ['chat_id']= channel_conf.chat_id
+os.environ['user_id'] = channel_conf.user_id
+
+# Declaring class Style
+class Style():
+    """
+    Declaring Style class
+    """
+    BLACK = '\033[30m'
+    RED = '\033[31m'
+    GREEN = '\033[32m'
+    YELLOW = '\033[33m'
+    BLUE = '\033[34m'
+    MAGENTA = '\033[35m'
+    CYAN = '\033[36m'
+    WHITE = '\033[37m'
+    UNDERLINE = '\033[4m'
+    RESET = '\033[0m'
+
+
+async def validate_message_with_sqs(queue_url, message_cloudwatch):
```

Comment:
This method looks too long , it will be good if you can divide the same into smaller methods.

Reply:
This helper 'asyncio_helper` has been refactored the methods moved to `filter_message_helper`